### DISCUSSION TITLE
2007/04/09/%E8%B0%B7%E6%AD%8C%E6%8B%BC%E9%9F%B3%E8%BE%93%E5%85%A5%E6%B3%95

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#150] [Announcements]
# 2007/04/09/%E8%B0%B7%E6%AD%8C%E6%8B%BC%E9%9F%B3%E8%BE%93%E5%85%A5%E6%B3%95

前几天，早上刚到公司，旁边的 Google 粉丝就对我们宣布，Google 推出拼音输入法了。我一向很欣赏 Google 的产品。所以马上要来试用。

https://blog.qizhen.xyz/2007/04/09/%E8%B0%B7%E6%AD%8C%E6%8B%BC%E9%9F%B3%E8%BE%93%E5%85%A5%E6%B3%95

**Comment by ruanqizhen** [comment]
之前博客下的留言：


sandy
2007年04月18日 13:01 编辑
厉害

回复
永青
2007年04月9日 22:11 编辑
大牛就是大牛，一个破拼音输入法还能写出这么长的文章。。。

回复
杰
2007年04月9日 19:00 编辑
我一直用的是紫光。
感觉还行的，关键是敲文字少，一般都是敲E文。
所以感觉华宇还是可以的。至少比只能ABC好多了。
整句输入我还真的没有试过呢。刚才这句是用紫光整句输入的。嘿嘿。

回复
俊
2007年04月9日 15:17 编辑
兄弟啊，牛文哦，偶也是第一时间用上谷歌拼音的，使用拼音输入法的历史也基本和您文章所述雷同。牛文，严重支持下。

回复
